assets/src/blocks/Spreadsheet/ArrowIcon.js

```diff
@@ -0,0 +1,3 @@
+export const ArrowIcon = () => (
```

Comment:
Can we use an SVG instead ?

Reply:
I don't think we have the appropriate loaders for this because webpack throws an error if you add it to the image folder and try to load it from there into a JS file.

Reply:
I see! Just an idea.. what if we add it within the stylesheet? I think it's great if we remove that file and make more consistent the use of SVG or any other image. Also reducing the bundle size which is not really significant
```jsx
<div className="arrow-icon"/>
```
```css
.arrow-icon {
    background: ...
}
```
Or th button:after {
background: ...
}